metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage_v2.py

```diff
@@ -247,17 +247,16 @@ def _populate_external_lineage_map(self, discovered_tables: List[str]) -> None:
         with PerfTimer() as timer:
             self.report.num_external_table_edges_scanned = 0
 
-            if self.report.edition == SnowflakeEdition.STANDARD:
```

Comment:
Is copy history available for all editions?

Reply:
yes